src/plugins/discover/public/application/main/hooks/use_discover_state.ts

```diff
@@ -218,7 +218,7 @@ export function useDiscoverState({
    */
   const onChangeDataView = useCallback(
     async (id: string) => {
-      const nextDataView = await dataViews.get(id);
+      const nextDataView = await dataViews.get(id, false);
```

Comment:
Did you try catching errors here?

Reply:
@mattkime Updated by wrapping with try/catch.